Sources/MockoloFramework/Parsers/SwiftSyntaxExtensions.swift

```diff
@@ -699,6 +710,8 @@ final class EntityVisitor: SyntaxVisitor {
         self.declType = declType
         super.init(viewMode: .sourceAccurate)
     }
+    
+    private static var ifConfigDeclCount: Int = 0
```

Comment:
This is not concurrency safe. There will be times when the same value is picked and becomes randomly order.